.bazelci/presubmit.yml

```diff
@@ -565,6 +572,8 @@ tasks:
       - "//..."
     test_targets:
       - "//..."
+    coverage_targets:
```

Comment:
I think you'll need to add `coverage_flags` here.

Reply:
😓 I thought `test_flags` are inherited as part of coverage?

on main, without my change, `bazel coverage` is broken for `Build with no_std + alloc using cc_common.link infrastructure for linking`

Reply:
This is true of `.bazelrc` flags but I don't think bazel CI works that way. Perhaps I'm misinterpreting https://github.com/bazelbuild/continuous-integration

Reply:
are there `coverage flags`? https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md

I don't see any mention of that?

Reply:
Coverage is not very well documented in the Bazel ecosystem. I remember seeing this deep in the python code some time ago but don't remember exactly where.